assets/pages/bundles-landing/bundlesLanding.jsx

```diff
@@ -21,8 +21,13 @@ import { trackOphan } from '../../helpers/abtest';
 
 // ----- Redux Store ----- //
 
-const store = pageInit(reducer);
-
+/* eslint-disable no-underscore-dangle */
+const store = pageInit(
+  reducer,
+  null,
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
```

Comment:
Out of curiosity, are there any performance implication for having this running in PROD?

Reply:
There should be no performance implications if the user doesn't have the plugin.  And I hope negligible if they do.